packages/app/lib/utils/index.js

```diff
@@ -31,6 +31,41 @@ class FirebaseUtilsModule extends FirebaseModule {
     return this.native.isRunningInTestLab;
   }
 
+  get androidPlayServices() {
+    if (isIOS) {
+      return {};
```

Comment:
```suggestion
      return {
        isAvailable: true,
        status: 0,
      };
```

Default value for iOS?

Reply:
Good catch, couldn't find it in v6 docs so didn't want it to break hence returning `{}`.